java/src/jmri/managers/DefaultShutDownManager.java

```diff
@@ -185,8 +185,13 @@ private boolean runShutDownTasks(boolean isParallel) {
                         log.info("Program termination aborted by \"{}\"", task.getName());
                         return false;  // abort early
                     }
-                } catch (Throwable e) {
+                } catch (Exception e) {
                     log.error("Error during processing of ShutDownTask \"{}\"", task.getName(), e);
+                } catch (Throwable e) {
+                    log.error("Unrecoverable error during processing of ShutDownTask \"{}\"", task.getName(), e);
+                    log.error("Terminating abnormally");
+                    // forcable halt
+                    System.exit(1);
```

Comment:
At this point, nothing we do (even sending to System.err) is ensured to work, but I'll add it in. Thanks for the suggestion.